src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  margin-top: 40px;
+  width: $width;
+  gap: $gap;
+  padding: $padding;
+
+  &__block {
+
+    width: $size;
+    height: $size;
+    border: 1px solid black;
+    background-color: #eee;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index}) {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+
+        &::before {
+          content: "#{$index}";
+        }
+
+        &:hover {
+
+          transform: translate(0, -20px);
+          transition: 0.5s;
```

Comment:
transition should work correct in both ways

Reply:
done